FrontBundle/Resources/config/routing.yml

```diff
@@ -23,3 +23,15 @@ services:
             - @open_orchestra_model.repository.site
         tags:
             - { name: routing.loader }
+
+    open_orchestra_front.router.file:
+        class: %open_orchestra_front.router.file.class%
+        arguments:
+            - @service_container
+            - %router.resource%
+            - { cache_dir: %kernel.cache_dir%, debug: %kernel.debug%, generator_class: %open_orchestra_front.router.file.options.generator_class%, generator_base_class: %open_orchestra_front.router.file.options.generator_base_class%, generator_cache_class: 'ProjectOpenOrchestraUrlGenerator', generator_dumper_class: %open_orchestra_front.router.file.options.generator_dumper_class%, matcher_class: %router.options.matcher_class%, matcher_base_class: %router.options.matcher_base_class%, matcher_dumper_class: %router.options.matcher_dumper_class%, matcher_cache_class: %router.options.matcher.cache_class% }
```

Comment:
I think it would be more readable if you use the multi-line syntax for this array